schemas/+lab/Location.m

```diff
@@ -3,7 +3,9 @@
 # This could be a room, a rig or a bench.
 location:                   varchar(32)
 -----
+-> lab.AcquisitionType
 location_description='':    varchar(255)
+
```

Comment:
I specified the acquisition type as a new table as you suggested.
I think this design is cleaner and better suited in the long term.
(In case a location is not matched anymore to a certain acquisition_type, we could use this key in the acquisition table)